CONTRIBUTING.md

```diff
@@ -298,7 +298,7 @@ To manually create a backport pull request, follow [the standard pull request pr
 > In the case of squashed or rebased merges, the commit hash will change and the new commits can be found in the merge message at the bottom of the master pull request.
 
 - In the pull request description, link to the original pull request to `master`.
-  The pull request title should include `[YY.MM]` matching the release you're backporting to.
+  The pull request title should include `[Backport release-YY.MM]` matching the release you're backporting to.
```

Comment:
I fail to see the value here. Also, this doesn't take into account the option to backport against stable staging branches.

Reply:
Fixed it so it also mentions the staging branches.

Reply:
Do you still fail to see the value of having the contributing guide suggesting to use the same PR title format as the automated GitHub Action?

Reply:
How about doing it the other way, making the backport action use the `[23.11]` format? It's not trivial to implement technically, but much easier socially, because not everybody will have to change their habits.

Reply:
I don't think there are that many manual backports being down for it to be an issue, also, I personally prefer having "backport" in the title.